src/connectedmachine/HISTORY.rst

```diff
@@ -2,6 +2,11 @@
 
 Release History
 ===============
+2.0.0b1
```

Comment:
the version after `1.0.0b1` should be `1.0.0` or `1.0.0b2`

Reply:
fixed